assets/src/edit-story/components/panels/document/slug/slug.js

```diff
@@ -38,41 +43,57 @@ export const MIN_MAX = {
   },
 };
 
-const BoxedTextInput = styled(TextInput)`
-  padding: 6px 6px;
-  border-radius: 4px;
-  flex-grow: 1;
-  &:focus {
-    background-color: ${({ theme }) => theme.DEPRECATED_THEME.colors.fg.white};
+const PermalinkRow = styled(Row)`
+  margin-bottom: 12px;
+
+  input {
+    color: ${({ theme }) => theme.colors.fg.tertiary};
+
+    :active,
+    :focus,
+    .${ThemeGlobals.FOCUS_VISIBLE_SELECTOR} {
```

Comment:
Just noticed there seems to be something odd happening with the link colours when tabbing into the link and hovering:
![colours](https://user-images.githubusercontent.com/3294597/111508402-f523e780-874b-11eb-9bd5-16f023f18b71.gif)

Reply:
Seems to come from the WordPress admin CSS:
<img width="356" alt="Screenshot 2021-03-17 at 19 11 42" src="https://user-images.githubusercontent.com/3294597/111508678-4338eb00-874c-11eb-921a-bda4963b750b.png">

Reply:
man, common css is a steady stream of surprises. :( We should fix this at the root in the Link. 😞

Reply:
I can fix that!